tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(SOF_FUZZER C)
+
+include(ExternalProject)
+
+set(parser_src_dir "${PROJECT_SOURCE_DIR}/../tplg_parser")
+set(parser_install_dir "${PROJECT_BINARY_DIR}/sof_parser/install")
+
+ExternalProject_Add(sof_parser_ep
+	SOURCE_DIR "${parser_src_dir}"
+	PREFIX "${PROJECT_BINARY_DIR}/sof_parser"
+	BINARY_DIR "${PROJECT_BINARY_DIR}/sof_parser/build"
+	CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${parser_install_dir}
+		-DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
+	BUILD_COMMAND ${CMAKE_COMMAND}
+	BUILD_ALWAYS 1
+	BUILD_BYPRODUCTS "${parser_install_dir}/lib/libsof_tplg_parser.so"
+)
+
+add_library(sof_parser SHARED IMPORTED)
+set_target_properties(sof_parser PROPERTIES IMPORTED_LOCATION "${parser_install_dir}/lib/libsof_tplg_parser.so")
+add_dependencies(sof_parser sof_parser_ep)
+
+add_executable(sof-fuzzer
+	main.c
+	qemu-bridge.c
+	topology.c
+	platform/byt-host.c
+)
+
+find_library(TPLG_PARSER sof_tplg_parser)
+if(TPLG_PARSER)
+	target_link_libraries(sof-fuzzer PRIVATE ${TPLG_PARSER})
```

Comment:
You should be able to just use `sof_parser` instead of `${TPLG_PARSER}` also find_library is not needed then.